resources/airflow.rb

```diff
@@ -1027,6 +1027,25 @@ def self.create_return_air_duct_zone(model, air_loop_name, adiabatic_const)
     return ra_duct_zone
   end
 
+  def self.create_sens_lant_load_actuator_and_equipment(model, name, space, is_latent, is_outside = false)
```

Comment:
"lant" should be "lat"?

Reply:
You are reusing existing code from the duct model to create the actuated objects. Excellent! 👍